src/main/java/com/example/java_pro_homework_7/Pet/Cat.java

```diff
@@ -0,0 +1,59 @@
+package com.example.java_pro_homework_7.Pet;
+
+public class Cat extends Animals {
+
+    public Cat(String name, String color, String breed) {
+        super(name, color, breed);
+        limitDistanceSwim = 0;
+        limitDistanceRun = 200;
+
+        totalCat();
+
+
+    }
+
+    static public int totalCat;
```

Comment:
Class' members should have a specific order:

constants
static variables
instance variables
constructors
getters + setters
public methods
private methods
Please change this class' members order according to the info in this comment.